src/status_im/ui/screens/multiaccounts/recover/subs.cljs

```diff
@@ -0,0 +1,53 @@
+(ns status-im.ui.screens.multiaccounts.recover.subs
```

Comment:
this file looks weird, have you seen this https://github.com/status-im/status-react/blob/develop/doc/codebase-structure-and-guidelines.md#subscriptions ?  you could have only one subscription :multiaccounts/recover and just retrieve keys in a view

Reply:
ok, i've deleted that file and now using root subscription